bin/prism

```diff
@@ -246,6 +248,24 @@ module Prism
       pp prism
     end
 
+    # bin/prism benchmark [source_file]
+    def benchmark(argv)
+      require "benchmark/ips"
+      require "parser/current"
+      require "ruby_parser"
+
+      filepath = argv.fetch(0) { File.expand_path("../lib/prism/translation/parser/compiler.rb", __dir__) }
+
+      Benchmark.ips do |x|
+        x.report("Parser::CurrentRuby") { Parser::CurrentRuby.parse_file(filepath) }
+        x.report("Parser::Prism") { Prism.parse_file(filepath) }
```

Comment:
This should be labeled just `Prism` as there is no `Parser::Prism` (it confused me for the results in the PR description).
I was about to make a PR but @kddnewton already fixed it in 40059d3400dc5f6c7ef0360993276637dfe70903 :)

Reply:
Ah dang, that's copy-pasta. Thanks for the note, Benoit